src/app/assets/languages/en.json

```diff
@@ -349,7 +349,7 @@
           "striping": "Lock striping",
           "striping-tooltip": "Uses a shared pool of locks for all entries in the cache. Striping lowers the memory footprint for locks but can reduce concurrency. If you disable striping, a lock is created for each entry in the cache.",
           "index-tuning": "Index tuning",
-          "index-tuning-tooltip": "Control how {{brandname}} interacts with an index to optimize performance.",
+          "index-tuning-tooltip": "To optimize cache performance, specify the methods that {{brandname}} uses to interact with the index.",
```

Comment:
I get that you're flipping sentences to put emphasis on the perceived value or user motivation but I think it's adding unnecessary verbiage.

Reply:
Yes, I made these changes based on [PatternFly best practices](https://www.patternfly.org/v4/ux-writing/ux-writing-best-practices#lead-with-the-benefit), and how my team generally writes. But, of course, these suggestions are optional. Because I don't have full context on this tool, you all will know better than I do if they don't add value or if they make the interface more confusing. These can easily be left unchanged if that's what you'd like!

Reply:
Thanks for linking to the PatternFly best practices @edonehoo